quality-control/human-clinical-and-health-data.md

```diff
@@ -1,27 +1,79 @@
+
+  Continuing with the example above, the ISIN register is used in the Czech Republic to obtain epidemiological data on the occurrence of patients with infectious diseases (including patients with COVID-19).  It is a universal system whose purpose is to obtain information on the occurrence of infectious diseases to assess the development of the epidemiological situation in the Czech Republic, to monitor the health status of the population and to manage the provision of health care. Data from regional hygiene stations, laboratories and hospitals caring for hospitalised patients were centralised in the ISIN during the SARS-CoV-2 pandemic. The ISIN system is managed by The Institute of Health Information and Statistics of the Czech Republic, an organisational unit of the state and has been established by the Ministry of Health of the Czech Republic.
+
+* Sample related data: Each sample should have a unique ID for its identification. Similarly, this ID would be the one used to link the sample data to the human clinical data. Thus, it is important to control this process to make sure the linkage is accurate. In addition, the quality dimensions mentioned before should be controlled also for these data.
+
+  In the collection point module of the CovIT from the Czech Republic, the unique ISIN request number is generated using online communication with ISIN and a unique barcode containing ISIN request number is generated and can be print on the label together with name, surname, personal identification number and date and time of sampling. Correct identification and processing of the sample in the laboratory is ensured by this labelling. The second module of CovIT software is a module for SARS-CoV-2 testing laboratories. After delivery of the sample to the laboratory, it can be registered through barcode scanning as well as unique ISIN request number or personal ID filling. All data are transferred from the collection point module which ensures the data accuracy. All important parameters related to the sample identification, storage (ID and position of the plate) and results are collected in the laboratory module.
+
+  In addition, the IMTM developed a registration portal for the samples that were collected out of the collection points and self-sampled by gargling using Gargtest self-sampling device. The unique collection tube barcode, personal data and sampling date were registered through a web portal, unique ISIN request number was generated, and all data were automatically transferred to CovIT software. The same portal was also used to retrieve the results of SARS-CoV-2 testing and self-reporting of risky contacts (based on unique code generated by CovIT and obtained by SMS).
```

Comment:
In addition, the IMTM developed a registration portal for the samples that were not collected at the collection points and self-sampled by gargling using a Gargtest self-sampling device. The unique collection tube barcode, personal data and sampling date were registered through a web portal, a unique ISIN request number was generated, and all data were automatically transferred to CovIT software. The same portal was also used to retrieve the results of SARS-CoV-2 testing and self-reporting of risky contacts (based on unique code generated by CovIT and obtained by SMS).